.travis.yml

```diff
@@ -3,7 +3,7 @@ language: node_js
 node_js:
   # we recommend testing addons with the same minimum supported node version as Ember CLI
   # so that your addon works for all apps
-  - '6'
+  - '8'
```

Comment:
why did you change the Node.js version for TravisCI here?

Reply:
Because the old version of node is causing issues when testing floating dependencies. We're pulling in packages that don't support 6 in their engines anymore, so we have to do this anyway.

We've updated a few other PRs (yet unmerged) in this repo to use Node 8 in the .travis.yml. This PR will just preempt them.

Reply:
then we should update https://github.com/rwjblue/ember-test-waiters/blob/master/package.json#L73 and declare it officially as a breaking change

Reply:
We don't want to do that yet until Node 8 is EOL, which is 12/31/19 (soon). Once that happens we will bump the engines version and major bump. We just wanted to avoid two major version bumps in succession. 

Hope this makes sense.

Reply:
yeah! but then we should keep testing on Node 6

Reply:
So as I said, if we keep Node 6 as the target for travis, when the floating dependencies test runs, the `yarn install` fails due to some packages requiring Node 8 and above. This would have prevented us from merging this PR, as CI would have failed. Perhaps there's a better way to address this. What did you have in mind?

Reply:
we could use Yarn resolutions to avoid this issue. not perfect, but better than not running CI on Node 6 at all.

Reply:
This seems like a reasonable alternative, but I want to emphasize that we're not planning on releasing until Node 8 is EOL. The current release is still tested on Node 6, and will continue to work for those users using Node 6.

I can totally use resolutions instead for this, but I'm not sure it's worth it at this point. If you feel strongly about it, I'm happy to open a new PR that updates both the travis and github actions implementations to use Node 6, and force the offending package to a version that supports Node 6 via resolutions. Let me know.

Reply:
if that is the case then we should update the package.json file to reflect those new minimum requirements on master

Reply:
Yep that's fine. I can do that.